### PR TITLE
[Model Update]: Battery Pass

### LINKED ISSUE #194: [Model Update]: Battery Pass in correspondence with Digital Product Pass
## Update Reason
Update of: https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.battery.battery_pass
to fit with the: #183 new Digital Product Passport

Additional issues to fix:
- fix all preferred naem
- power fade wrong preferred name and description
- stateOfHealth: description doens't fit to expected input
- batteryType and batteryModel describe the same thing with the same example
- also add and closes #81

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists
- [x] The potential updates are discussed with all stakeholders
- [x] The potential update will be either 
  - [ ] backward compatible or 
  - [x] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed

### PR DESCRIPTION
**Pull request by catroest**
## Description
Fundamental changes to be conform with the latest release of the regulation.

 -->

Closes #194 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.2)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [ ] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by catroest** [comment]
valid with the merge of #498 as it includes this version in several imports

**Comment by catroest** [comment]
@agg3fe NOT ready for MS2

**Comment by catroest** [comment]
@tom-rm-meyer-ISST please MS2 review. The model is valid, after the merge of the Digital Product Passport PR.

File: io.catenax.battery.battery_pass/4.0.0/BatteryPass.ttl [hunk 1]
```diff
@@ -0,0 +1,1180 @@
...
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.battery.digital_product_passport:4.0.0#> .
```
Comment (agg3fe) [inline io.catenax.battery.battery_pass/4.0.0/BatteryPass.ttl:30]: wrong name specified

File: io.catenax.battery.battery_pass/4.0.0/BatteryPass.ttl [hunk 2]
```diff
@@ -0,0 +1,1180 @@
...
+   samm:characteristic :CycleLifeCharacteristic .
+
+:percentValue a samm:Property ;
+   samm:preferredName "Percent Value"@en ;
+   samm:description "A value as a percentage in the form of a double."@en ;
+   samm:characteristic :PercentMeasurement ;
+   samm:exampleValue "50.0"^^xsd:double .
+
+:dataRecordTime a samm:Property ;
+   samm:description "The entry date and time corresponding to when a specific parameter was measured. This attribute is mentioned in the Battery regulation 2023/1542 from 12 July 2023 in Annex XIII (4):\n(d) information and data resulting from its use, including the number of charging and discharging cycles and negative events, such as accidents, as well as periodically recorded information on the operating environmental conditions, including temperature, and on the state of charge."@en ;
```
Comment (agg3fe) [inline io.catenax.battery.battery_pass/4.0.0/BatteryPass.ttl:941]: `preferredName` is missing

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by jacewski-bosch** [comment]
All required reviewers have approved this PR (see reviewers section) --> OK
 The new aspect (version) will be implemented by at least one data provider  --> OK
 The new aspect (version) will be consumed by at least one data consumer --> OK
 There exists valid test data --> OK
 In case of a new (incompatible) major version to an existing version, a migration strategy has been developed --> OK
 The model has at least version '1.0.0'  --> OK
 If a previous model exists, model deprecation has been checked for previous model --> OK
 The release date in the Release Note is set to the date of the MS3 approval --> OK

MS3 approval given by the group

@agg3fe please merge

**Pull request merged**